src/__tests__/config-loader.test.ts

```diff
@@ -102,15 +88,14 @@ describe("config-loader", (): void => {
     const result = configLoader({
       explicitParams: undefined,
       cwd: "/baz",
-      // tslint:disable-next-line:no-any
-      tsConfigLoader: (_: any) => ({
+      tsConfigLoader: () => ({
         tsConfigPath: "/baz/tsconfig.json",
         baseUrl: "/baz",
         paths: {},
       }),
     });
 
     const successResult = result as ConfigLoaderSuccessResult;
-    assert.equal(successResult.absoluteBaseUrl, "/baz");
+    expect(successResult.absoluteBaseUrl).toEqual("/baz");
```

Comment:
This was the failing test from https://github.com/dividab/tsconfig-paths/pull/174.